test/runtests.jl

```diff
@@ -1,7 +1,33 @@
+
 using Test
+urls = Vector{AbstractString}(["https://docs.julialang.org/en/v1/"])
```

Comment:
It's great that you're adding some test! It would make sense to add the Github CI to run every time you make a PR.

On the organization, created separate files in `test/` folder that match the `src/` script file you're testing. That makes it very easy to navigate for others.